streaming/base/partition/orig.py

```diff
@@ -52,6 +52,9 @@ def get_partitions_orig(num_samples: int,
 
     batch_size = batch_size or 1
 
+    # If drop_first isn't a multiple of num_physical_nodes, round down to make it divisible.
+    drop_first -= drop_first % num_physical_nodes
```

Comment:
If the drop_first isn't a multiple of num_physical_nodes, does this mean that there would a certain group of sample(s) that would be repeated (before checkpoint and after checkpoint) ?  If yes, then can we add a logger.warning message saying that some sample(s) will be repeated ?

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/301/commits/86b150a00025a92f1db686b0c85732c3f50591a5.